__tests__/app.test.js

```diff
@@ -148,9 +148,53 @@ describe("app", () => {
         .get("/api/reviews/999")
         .expect(404)
         .then(({ body }) => {
-          console.log(body.msg);
           expect(body.msg).toBe("Not Found");
         });
     });
   });
 });
+describe("/api/reviews/:review_id/comments", () => {
+  it("200: GET: it should respond with an object of comments for the given review_id", () => {
+    return request(app)
+      .get("/api/reviews/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toBeInstanceOf(Object);
+      });
+  });
+  it("200:GET: should respond with an empty array if review_id has no comments", () => {
+    const reviewId = 100;
```

Comment:
Does this review_id exist? if not then I would expect this to 404, 
What we want to get from this is that 
- if the review exists but doesnt have any comments we want a 200 and an empty array
- If the review doesn't exist then it should 404